core/settings.py

```diff
@@ -109,7 +109,7 @@ def get_env_variable(var_name, required=True, type=str):
 def floc_middleware(get_response):
     def middleware(request):
         response = get_response(request)
-        response["Permissions-Policy"] = "interest-cohort=()"
+        response["Referrer-Policy"] = "no-referrer"
```

Comment:
Note: I removed the google FLOC header as it's now obsolete, and replaced it with a no-referer policy in order to secure navigation to external hosts.